extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -385,9 +385,60 @@ describes.realWin(
             'https://server-test-3/':
               '{"consentRequired": true, "consentStateValue": "unknown"}',
             'https://geo-override-check2/': '{"consentRequired": true}',
+            'https://gdpr-applies/':
+              '{"consentRequired": true, "gdprApplies": false}',
           };
         });
 
+        describe('gdprApplies value', () => {
+          it('uses given value', async () => {
+            const inlineConfig = {
+              'consentInstanceId': 'abc',
+              'consentRequired': 'remote',
+              'checkConsentHref': 'https://gdpr-applies/',
+            };
+            ampConsent = getAmpConsent(doc, inlineConfig);
+            await ampConsent.buildCallback();
+            await macroTask();
+            const stateManagerGdprApplies = await ampConsent
+              .getConsentStateManagerForTesting()
+              .getConsentInstanceGdprApplies();
+            expect(stateManagerGdprApplies).to.be.false;
```

Comment:
Could you add one more test, that the gdprApplies value resolves when the `checkConsentHref` is not defined.